javascript/src/parser.ts

```diff
@@ -195,45 +193,11 @@ export class Parser {
    * @param threshold The threshold score to control the granularity of chunks.
    */
   applyElement(parentElement: HTMLElement, threshold = DEFAULT_THRES) {
-    parentElement.style.wordBreak = 'keep-all';
-    parentElement.style.overflowWrap = 'break-word';
-    const chunks = this.parse(parentElement.textContent || '', threshold);
-    let charsToProcess = chunks.join(SEP);
-    const ownerDocument = parentElement.ownerDocument;
-
-    const processChildren = (parent: HTMLElement) => {
-      const toSkip = SKIP_NODES.has(parent.nodeName);
-      const children = [...parent.childNodes];
-      for (const child of children) {
-        if (child.nodeType === NODETYPE.TEXT) {
-          let textNodeContent = '';
-          const nodesToAdd: (HTMLElement | Text)[] = [];
-          (child.textContent || '').split('').forEach(char => {
-            if (toSkip) {
-              textNodeContent += char;
-              charsToProcess = charsToProcess.slice(
-                charsToProcess[0] === SEP ? 2 : 1
-              );
-            } else if (char === charsToProcess[0]) {
-              textNodeContent += char;
-              charsToProcess = charsToProcess.slice(1);
-            } else if (charsToProcess[0] === SEP) {
-              nodesToAdd.push(ownerDocument.createTextNode(textNodeContent));
-              nodesToAdd.push(ownerDocument.createElement('wbr'));
-              charsToProcess = charsToProcess.slice(2);
-              textNodeContent = char;
-            }
-          });
-          if (textNodeContent) {
-            nodesToAdd.push(ownerDocument.createTextNode(textNodeContent));
-          }
-          child.replaceWith(...nodesToAdd);
-        } else if (child.nodeType === NODETYPE.ELEMENT) {
-          processChildren(child as HTMLElement);
-        }
-      }
-    };
-    processChildren(parentElement);
+    const html_processor = new HTMLProcessor(this, {
```

Comment:
nit. Let's rename this to `htmlProcessor` just for naming consistency :)

Reply:
Done, sorry I was using wrong styleguide ;-P